modules/machineset-modifying.adoc

```diff
@@ -8,13 +8,23 @@
 [id="machineset-modifying_{context}"]
 = Modifying a compute machine set by using the CLI
 
+You can modify the configuration of a compute machine set and then propagate the changes to the machines in your cluster by using the CLI.
+
+By updating the compute machine set configuration, you can enable features or change the properties of the machines it creates.
 When you modify a compute machine set, your changes only apply to compute machines that are created after you save the updated `MachineSet` custom resource (CR).
 The changes do not affect existing machines.
-You can replace the existing machines with new ones that reflect the updated configuration by scaling the compute machine set.
+
+[NOTE]
+====
+Changes made in the underlying cloud provider are not reflected in the `Machine` or `MachineSet` CRs.
+To adjust instance configuration in cluster-managed infrastructure, use the cluster-side resources.
+====
+
+You can replace the existing machines with new ones that reflect the updated configuration by scaling the compute machine set to create twice the number of replicas and then scaling it down to the original number of replicas.
 
 If you need to scale a compute machine set without making other changes, you do not need to delete the machines.
 
-[NOTE]
+[IMPORTANT]
```

Comment:
The router and other OCP workloads that want to be highly available should have PodDisruptionBudget guards.  Requesting compute scale to zero would block on draining more than those PDBs allow.  Being up against the PDB guards for a while would cause alerts to fire, and let the cluster admin know they'd squeezed things too far.  So I think it's worth having this note, and I'm ok with it moving to `IMPORTANT`, but I don't want to imply "if you miss reading this note, you might break your router and web-console!".  We're just trying to save admins the hassle of only scaling down and then realizing (via the PDB alerts) that they want to scale back up.

Reply:
I think given that, NOTE is more appropriate :) thanks!